source/adapters/cuda/image.cpp

```diff
@@ -780,6 +780,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageGetInfoExp(
     case 4:
       ChannelOrder = UR_IMAGE_CHANNEL_ORDER_RGBA;
       break;
+    default:
```

Comment:
Does this fail only on SYCL? The previous PR was built without issues. Is UR missing `-Wswitch-default` or something like that?

Reply:
The previous PR built without issues because image.cpp was missing from the CmakeLists. The error is a `[-Werror=maybe-uninitialized`  which can be seen failing here:

https://github.com/oneapi-src/unified-runtime/actions/runs/6326508321/job/17180224616

I  think that on sycl side, we were removing -werror for unified runtime.